packages/connect/e2e/tests/device/authenticateDevice.test.ts

```diff
@@ -56,7 +56,7 @@ describe('TrezorConnect.authenticateDevice', () => {
         },
     };
 
-    it('validation successful', async () => {
+    conditionalTest(['!T2T1'], 'validation successful', async () => {
```

Comment:
I keep forgetting how is conditionalTest supposed to be used. Did you try it locally that it behaves as expected?

Reply:
I just tested and with this `!T2T1` those test do not run when device model is `T2T1`.

So if you run the command below those tests are skipped but if you replace T2T1 with T3B1 then they run successfully:

```bash
./docker/docker-connect-test.sh node -p "authenticateDevice" -f "2-main" -c -m T2T1 -t 2.0.33
```

Reply:
Yes, it works locally. But CI skips it because it runs all tests on T2T1, which is the default.